README.md

```diff
@@ -12,7 +12,7 @@ A Client library for OAuth2 which uses JWT for the generation of access tokens.
               .subject("myapp ")
               .build();
 
-  TokenSource tokenSource = config.tokenSource();
+  TokenSource tokenSource = TokenSources.reusableTokenSource(new OkHttpTokenSource(config));
```

Comment:
This is how the API looks now: 
```java
 OkHttpClient client = new OkHttpClient.Builder()
            .addInterceptor(
                    BearerAuthenticationInterceptor.newInterceptor(
                            JwtConfig.newConfig("email", "token url", key).build()
                    )
            )
            .build();
```

OkHttpTokenSource is marked as private class and is no longer publicly available.